notifications_utils/s3.py

```diff
@@ -82,6 +93,17 @@ def s3download(
         )
         s3 = session.resource("s3", config=AWS_CLIENT_CONFIG)
         key = s3.Object(bucket_name, filename)
+        # This 'proves' that use of moto in the relevant tests in test_send.py
+        # mocks everything related to S3.  What you will see in the logs is:
+        # Exception: CREATED AT <MagicMock name='resource().Bucket().creation_date' id='4665562448'>
+        #
+        # raise Exception(f"CREATED AT {_s3.Bucket(bucket_name).creation_date}")
+        if os.getenv("NOTIFY_ENVIRONMENT") == "test":
+            teststr = str(s3.Bucket(bucket_name).creation_date).lower()
+            if "magicmock" not in teststr:
+                raise Exception(
+                    f"xxxxxtest not mocked, use @mock_aws creation date is {teststr}"
```

Comment:
Same note as above:  it looks like this exception message is trying to say two different things; is this supposed to just be referencing the current test?  And if so, is there way to perhaps reference the current test this is running in?  `pytest` might provide a way to get that.

Reply:
Same comment as above.  The individual test will fail if not mocked because this exception will be thrown, so we we have a list of failed tests that are not mocked.  There is no need to reference the test from this exception message.